bin/eslint-formatters-calypso/src/lib/parsing-errors.js

```diff
@@ -0,0 +1,27 @@
+/* eslint strict: "off" */
+
+'use strict';
+
+module.exports = function( report ) {
+	// deep clone the report, so we can create a new one
+	// to tweak and edit in place - as this function remains pure.
+	const newReport = [];
+
+	report.forEach( file => {
```

Comment:
A lot more [beautiful and readable](https://github.com/nssw/eslines/blob/master/src/lib/parsing-errors.js).